index.html

```diff
@@ -64,6 +64,21 @@ <h1>Solid</h1>
 -->
 <!-- /headerwrap -->
 
+	    <div id="draft-warning"></div>
+<script>
+  if (document.location.hostname === 'localhost' || document.location.hostname === 'solid.github.io') {
+    const draftWarningElement = document.getElementById('draft-warning')
+    draftWarningElement.innerHTML = `
+      <div class="message is-danger is-large" role="alert">
+        <div class="message-body">
+          You are currently viewing the historical site of the Solid project which is was incepted at MIT. If you are looking for reliable information, visit the live site at <a href="https://solidproject.org/" title="The official Solid website">SolidProject.org</a>.

```

Comment:
Suggest a slight copy change:

You are viewing the historical site of the Solid MIT Project. Visit the <a href="https://solidproject.org">official Solid website</a> for reliable and up-to-date information.

Reply:
Perhaps:

You are viewing the original website of the Solid Project at MIT. Please join us at the official Solid project website for current activities.